toolchain/driver/compile_subcommand.cpp

```diff
@@ -410,6 +432,7 @@ class CompilationUnit {
     CARBON_CHECK(parse_tree_);
     return {
         .value_stores = &value_stores_,
+        .timings = &*timings_,
```

Comment:
```suggestion
        .timings = &timings_,
```

(note comment elsewhere about pointer)